### DISCUSSION TITLE
Assigning default team to new LDAP users?

### DISCUSSION THREAD

**Discussion by ghost** [DependencyTrack/dependency-track#629] [Q&A]
My question is: Is there a way to add new LDAP users to a team upon creation? I would like to assign the role of **User** to all LDAP Users, even the ones that log in for the first time.

LDAP is configured correctly - if a user logs in with their LDAP credentials, a new LDAP user is created for them in dependency-track. But new users are in 0 Teams.

I tried to configure the **Teams > LDAP Group Mapping**, but adding a group there didn't seem to have any effect at all.

**Comment by stevespringett** [comment]
Ensure the LDAP filters defined are correct. Refer to:
https://docs.dependencytrack.org/getting-started/ldap-configuration/

Also ensure that these two properties are set to true:
```
alpine.ldap.user.provisioning
alpine.ldap.team.synchronization
```
or if using Docker:
```
ALPINE_LDAP_USER_PROVISIONING
ALPINE_LDAP_TEAM_SYNCHRONIZATION
```

**Comment by ghost** [comment]
I double checked the filters, enabled synchronization (provisioning was already enabled), but new LDAP Users still have 0 Teams.

The group set in LDAP Group Mapping is the same group that is displayed for the LDAP Users, eg.:

LDAP User example:
cn=LastName\2C FirstName,ou=**ABC**,ou=Office,dc=net,dc=work

Team LDAP Group Mapping:
ou=**ABC**,ou=Office,dc=net,dc=work
